app/views/admin/take_part_pages/index.html.erb

```diff
@@ -1,38 +1,59 @@
-<% page_title 'Get involved' %>
+<% content_for :page_title, "Get involved" %>
+<% content_for :title, "Get involved" %>
+<% content_for :title_margin_bottom, 4 %>
 
-<div class="get-involved-header">
-  <h1>Get involved</h1>
-  <%= link_to "View on website", get_involved_url %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p class="govuk-body">
+      <%= link_to "View on website", get_involved_url, class: "govuk-link", target: "_blank" %>
```

Comment:
Does this need to live inside a row as it's full width anyway?

Reply:
I guess strictly speaking it doesn't, but it seems to be the established pattern to use `govuk-grid-column-full` inside `govuk-grid-row` where we have full width elements, so I'd be inclined to leave it like this here as well.